job-shop-scheduling/README.rst

```diff
@@ -1,8 +1,91 @@
-Demo of Job Shop Scheduling
-===========================
+Job Shop Scheduling Demo
+========================
+A demo on how to optimally schedule jobs using a quantum computer.
 
-Further Information
--------------------
+Given a set of jobs and a finite number of machines, how should you schedule
+your jobs on those machines such that all your jobs are completed at the
+earliest possible time? This question is the Job Shop Scheduling Problem!
+
+Each of our jobs can be broken down into smaller machine-specific tasks. For
```

Comment:
Your last sentence seems to say that the preceding paragraph defined JSS, so it will be unclear to readers what this paragraph is doing -- maybe open it by saying that you're giving a more detailed description